src/model/neighborlist_modes.h

```diff
@@ -0,0 +1,208 @@
+/*
+ * neighborlist_modes.h
+ *
+ *  Created on: Dec 4, 2017
+ *      Author: glenn
+ */
+
+#ifndef SRC_MODEL_NEIGHBORLIST_MODES_H_
+#define SRC_MODEL_NEIGHBORLIST_MODES_H_
+
+
+#include "neighborlist.h"
+
+
+/**
+ * this is an overloaded Version of the original  NLPotForce function which directly calculates the counteracting forces acting on the receptor.
+ */
+template<typename REAL>
+void NLPotForce(
+		NLGrid<REAL> const* grid,
+		Protein<REAL> const* rec, //TODO: use receptor positions instead of receptor
+		Protein<REAL> const* lig,
+		SimParam<REAL> const* simParam,
+		ParamTable<REAL> const* table,
+		REAL const* RecPosX,
+		REAL const* RecPosY,
+		REAL const* RecPosZ,
+		REAL const* LigPosX,
+		REAL const* LigPosY,
+		REAL const* LigPosZ,
+		REAL* outLig_fx,
+		REAL* outLig_fy,
+		REAL* outLig_fz,
+		REAL* outLig_E,
```

Comment:
Are you sure that outLig_E  and outRec_E must be differentiated? Is there not only one Energy in the system?

Reply:
while I was writing it i realized that that's most likely the case. Id agree